### DISCUSSION TITLE
Is it possible to publish bundle on form factor

### DISCUSSION THREAD

**Discussion by Themaapps** [Triple-T/gradle-play-publisher#1093] [General]
Hi team,

I am wondering if the library is supporting the new wear OS specific release track?
Google dev console will enforce this track for wear os releases on end of august.
It seems that the api is supporting tracks, with the [prefix]:defaultTrackName format, like "wear:production"
https://developers.google.com/android-publisher/tracks?hl=fr#ff-track-name

Will this do the trick?
track.set("wear:production")

Is it supported with the current version ? It is not mentionned in the Readme.

Thank you again for this great library :)

**Comment by Themaapps** [comment]
To give an answer to my question: the track "wear:production" is working :)

**Comment by JonAlexander-AT** [comment]
I'm running into an issue with release notes when I use a wear: prefix

> A failure occurred while executing com.github.triplet.gradle.play.tasks.PublishApk$Processor
   > Cannot convert URL 'wear:internal.txt' to a file.

Edit: Workaround mentioned here: https://github.com/Triple-T/gradle-play-publisher/issues/1088
